Resources/Prototypes/Entities/Mobs/NPCs/funnyayadoll.yml

```diff
@@ -0,0 +1,89 @@
+- type: entity
+  name: poltergeist
+  id: Mobayadoll
+  parent: SimpleMobBase
+  description: AWW its so cute.
+  components:
+   - type: Vocal
+    maleScream:
+      collection: doll
+    femaleScream:
+      collection: doll
+  - type: PsionicBonusChance
+      multiplier: 10
+      warn: false
+ - type: Carriable
+ - type: Loadout
+    prototypes: [PassengerGear]
+  - type: CombatMode
+   canDisarm: true
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 4.00
+    baseSprintSpeed : 7.00
+  - type: InputMover
+  - type: MobMover
+  - type: Reactive
+    groups:
+      Flammable: [Touch]
+      Extinguish: [Touch]
+  - type: Faction
+    factions:
+    - SimpleHostile
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/ayadoll.rsi/dollspritesfull.png
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: poltergeisst
+  - type: Physics
+    bodyType: KinematicController
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.7
+      density: 230
+      mask:
+      - SmallMobMask
+      layer:
+      - SmallMobLayer
+  - type: MobState
+    thresholds:
+      0: Alive
+      125: Critical
+      150: Dead
+  - type: MeleeWeapon
+    hidden: true
+    angle: 0
+    animation: WeaponArcclaw
+    damage:
+      types:
+        Slash: 5
+        Piercing: 15
+    baseDecayRate: 0.1
+  - type: Appearance
+  - type: DamageStateVisuals
+    rotate: true
+    states:
+      Alive:
+        Base: poltergeist
+      Critical:
+        Base: poltergeist_dead
```

Comment:
There is no corresponding image for this. Related to the other comments about the RSI.